src/app/handlers/command.handler.ts

```diff
@@ -14,30 +30,40 @@ export class CommandHandler extends Handler {
     super();
   }
 
-  public async execute(message: types.IMessage): Promise<void> {
-    const command = this.build(message.content);
+  public async execute(message: types.IMessage | CommandInteraction): Promise<void> {
     const plugins = this.container.pluginService.plugins;
     const aliases = this.container.pluginService.aliases;
 
-    // checks to see if the user is actually talking to the bot
-    if (!command) {
-      return;
-    }
-
-    if (message.mentions?.everyone) {
-      message.reply('You cannot use a plugin, while pinging everyone.');
-      return;
-    }
+    let plugin;
```

Comment:
do you need to declare this here? it seems like you can tightly scope it to each branch of the if/else

Reply:
Doen